app/models/court_case.rb

```diff
@@ -0,0 +1,35 @@
+class CourtCase
+  include ActiveModel::Serialization
+
+  attr_reader :id, :case_seq, :begin_date, :case_type, :case_info_number, :case_status, :agency_id
+
+  def build_from_nomis(court_case)
+
+    @case_info_number = court_case['caseInfoNumber']
+    @id = @case_info_number
```

Comment:
Not sure why this has a duplicate identifier? I'd be inclined to drop `case_info_number` completely from this model and rely solely on `id`. It will likely cause confusion and inconsistency having two attributes with the same purpose.

Reply:
Good point, In general ID is a UUID in our models/tables, but in this case we do not have a ID, because the court case is not persistent. That's why I'm setting it to the `case_info_number`. 

Not sure about removing the `case_info_number`, that would be unexpected by the FE (I think it's already using it)